tests/NHSD.BuyingCatalogue.Ordering.Api.IntegrationTests/Builders/OrderSummaryData.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+using NHSD.BuyingCatalouge.Ordering.Api.Testing.Data.Entities;
+
+namespace NHSD.BuyingCatalogue.Ordering.Api.IntegrationTests.Builders
+{
+    public class OrderSummaryData
+    {
+        public List<OrderEntity> OrderEntities { get; }
```

Comment:
Any reason the OrderSummaryData needs a list of orders? Could this just be a single order entity?